### PR TITLE
RDoc-1110 Studio > Database > Documents > Document View

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown [hunk 1]
```diff
@@ -1,4 +1,79 @@
...
+       _Node ID_ - uniquely identifies a node  
+       _etag_ - a 64 bit number that is incremented on every operation in a database  
+   * **Modified** - The last time the document was modified by any client  
+   * **Size** - The Document size (including attachments)  
+<br/>
+3. **Document content** - The document properties and values in JSON format  
+   * a. **Nested Data**  
+        * Another JSON object can be nested in a property value  
+<br/>
+   * b. **Included Documents**
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: Included -> referenced or related

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown [hunk 2]
```diff
@@ -1,4 +1,79 @@
...
+       _etag_ - a 64 bit number that is incremented on every operation in a database  
+   * **Modified** - The last time the document was modified by any client  
+   * **Size** - The Document size (including attachments)  
+<br/>
+3. **Document content** - The document properties and values in JSON format  
+   * a. **Nested Data**  
+        * Another JSON object can be nested in a property value  
+<br/>
+   * b. **Included Documents**  
+        * Your can reference other documents from other collections in a document property value
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: from other collections - you can reference from the same collection as well e.g. 

http://live-test.ravendb.net/studio/index.html#databases/edit?&collection=Employees&database=Northwind&id=employees/9-A

`"ReportsTo": "employees/5-A",`

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown [hunk 3]
```diff
@@ -1,4 +1,79 @@
...
+   * **Modified** - The last time the document was modified by any client  
+   * **Size** - The Document size (including attachments)  
+<br/>
+3. **Document content** - The document properties and values in JSON format  
+   * a. **Nested Data**  
+        * Another JSON object can be nested in a property value  
+<br/>
+   * b. **Included Documents**  
+        * Your can reference other documents from other collections in a document property value  
+        * These included documents can then be easily retrieved when a client _Loads_ the document from the server
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: Maybe: Such documents can be included in a single request to the server using Load + Include method (link https://ravendb.net/docs/article-page/4.0/csharp/client-api/session/loading-entities#load-with-includes)

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown [hunk 4]
```diff
@@ -1,4 +1,79 @@
...
+   * a. **Nested Data**  
+        * Another JSON object can be nested in a property value  
+<br/>
+   * b. **Included Documents**  
+        * Your can reference other documents from other collections in a document property value  
+        * These included documents can then be easily retrieved when a client _Loads_ the document from the server  
+        * e.g., In the example above, _'categories/1-A'_ document shows in the _'Category'_ property  
+<br/>
+   * c. **Metadata**  
+        * This is additional information that is unrelated to the document itself but is rather about the document
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: is unrelated to the document itself but is rather about the document - not sure what did you mean

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown [hunk 5]
```diff
@@ -1,4 +1,79 @@
...
+<br/>
+   * c. **Metadata**  
+        * This is additional information that is unrelated to the document itself but is rather about the document  
+        * The metadata, also in JSON format, is embedded inside the document and is an integral part of it  
+        * RavenDB server reserves metadata properties that start with _'@'_ for its own use  
+        * You can add properties to the metadata and use to store your own values  
+        * Note: only the following metadata properties will show in the Studio:  
+           * ***@collection*** - determines to which collection the document belongs to  
+           * ***@flags*** - i.e. if a document has attachments, revisions, etc.  
+           * Any ***client generated*** properties - any metadata properties generated in code by a client
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: maybe let's call it custom metadata provided by a user

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown [hunk 6]
```diff
@@ -1,4 +1,79 @@
...
+        * RavenDB server reserves metadata properties that start with _'@'_ for its own use  
+        * You can add properties to the metadata and use to store your own values  
+        * Note: only the following metadata properties will show in the Studio:  
+           * ***@collection*** - determines to which collection the document belongs to  
+           * ***@flags*** - i.e. if a document has attachments, revisions, etc.  
+           * Any ***client generated*** properties - any metadata properties generated in code by a client  
+<br/>
+
+4. **Related Document**  
+   * List of related documents - those are the documents that are _included_ inside the document. (see 3b above)
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: they are _not_ included

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown [hunk 7]
```diff
@@ -1,4 +1,79 @@
...
+1. **General** actions  
+   * Save document  
+   * Delete document  
+   * Clone document - Create a clone of the current document  
+     * A copy of the document without attachments is created  
+     * It can be saved with a new ID  
+<br/>
+2. **Copy** to clipboard  
+   * Copy to clipboard - The document content is copied  
+   * Copy as C# class - The c# entity class (on which the document json is based) is copied
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: c# -> C#,
on which the document json is based-> maybe: reflecting JSON content

File: Documentation/4.0/Raven.Documentation.Pages/studio/overview/overview.markdown [hunk 8]
```diff
@@ -1,41 +0,0 @@
-﻿## Studio Overview
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/overview/overview.markdown]: Did you remove it intentionally?

File: Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown [hunk 9]
```diff
@@ -1,4 +1,21 @@
 ﻿# Create A Database : General Flow
+---
+
+{NOTE: }
+
+From the Studio, the database creation options are:  
+
+* **Regular** database - see here below
```
Comment (arekpalinski) [inline Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: here below  -> here or below?

Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/server/databases/create-new-database/general-flow.markdown]: fixed to 'below'

File: Documentation/4.0/Raven.Documentation.Pages/studio/overview/overview.markdown (hunk 8 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/overview/overview.markdown]: 1. Yes, it should Not be here
2. It already exists directly under 'Studio' 
3. **Tree** plan can be seen in RDoc-738

Comment (arekpalinski) [reply Documentation/4.0/Raven.Documentation.Pages/studio/overview/overview.markdown]: Ok. Just wanted to make sure it was intentional removal

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown (hunk 7 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: done - thanks

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown (hunk 4 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: 1. Well.. this was taken from 'Book' - see beginning of chapter 2.7
2. Maybe just use:  *'This is additional information about the document'* ?

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown [hunk 10]
```diff
@@ -1,4 +1,79 @@
...
+* Actions such as cloning the document, deleting, adding attachments and much more can be performed.  
+{NOTE/}
+
+---
+
+{PANEL: The Document View}
+
+![Figure 1. Document View](images/document-view-1.png "Document: 'Suppliers/1-A' in the 'Suppliers' Collection")
+
+1. **Document Name**
```
[the remark this answers is not in the document]
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown [hunk 11]
```diff
@@ -1,4 +1,79 @@
...
+
+---
+
+{PANEL: The Document View}
+
+![Figure 1. Document View](images/document-view-1.png "Document: 'Suppliers/1-A' in the 'Suppliers' Collection")
+
+1. **Document Name**  
+   * For detailed explanation about the possible identifiers See [Create New Document](../../../../todo-update-me-later)  
+   * Clicking the link, right by the name, will show the _raw document output_ that is received  from the server
```
[the remark this answers is not in the document]
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown (hunk 4 above)
Comment (arekpalinski) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: Sure

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown [hunk 12]
```diff
@@ -1,4 +1,79 @@
...
+   * For detailed explanation about the possible identifiers See [Create New Document](../../../../todo-update-me-later)  
+   * Clicking the link, right by the name, will show the _raw document output_ that is received  from the server  
+<br/>
+2. **Document properties**  
+   * **Change-Vector**  
+     * The _change-vector_ uniquely marks the specific version of the document globally in the cluster  
+     * Each time a document is modified, a new _change-vector_ is generated  
+     * Used for optimistic  concurrency control, various internal operations and caching  
+     * It is composed of a list of node _IDs_ and _etags_  
+       _Node ID_ - uniquely identifies a node
```
[the remark this answers is not in the document]
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: 1. Was taken from the book - chapter 3.6.3
2. Fixed - thanks

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown [hunk 13]
```diff
@@ -1,4 +1,79 @@
...
+<br/>
+2. **Document properties**  
+   * **Change-Vector**  
+     * The _change-vector_ uniquely marks the specific version of the document globally in the cluster  
+     * Each time a document is modified, a new _change-vector_ is generated  
+     * Used for optimistic  concurrency control, various internal operations and caching  
+     * It is composed of a list of node _IDs_ and _etags_  
+       _Node ID_ - uniquely identifies a node  
+       _etag_ - a 64 bit number that is incremented on every operation in a database  
+   * **Modified** - The last time the document was modified by any client
```
[the remark this answers is not in the document]
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown (hunk 2 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: done

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown (hunk 5 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: ok

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown (hunk 6 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: Fixed to:
*List of related documents - those are the documents that are **_referenced_** inside the document*

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown (hunk 1 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: ok - Referenced

File: Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown (hunk 3 above)
Comment (Danielle9897) [reply Documentation/4.0/Raven.Documentation.Pages/studio/database/documents/document-view.markdown]: great - thanks

**Pull request merged**